deployment/deploy-prod.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+# Promote App Engine
+- name: gcr.io/cloud-builders/gcloud
+  args: ['app', 'versions', 'migrate', '--quiet', '$COMMIT_SHA', '--service=default']
```

Comment:
Question: whereabouts do App Engine versions get named? I don't believe the current versions are based off a commit SHA, and I wanted to confirm that the naming has been changed somewhere so that they are, otherwise I don't think this is going to work as intended?

i.e.

```
$ gcloud --project oss-vdb app versions list --service=default --hide-no-traffic
SERVICE  VERSION.ID                                TRAFFIC_SPLIT  LAST_DEPLOYED              SERVING_STATUS
default  9805438dc8731dabd78c8b3eb6871cb12a5090b5  1.00           2023-02-28T13:35:22+11:00  SERVING
```

doesn't seem to correlate with a commit hash I can find on GitHub...

Reply:
The versions are named in the App Engine deploy script (which, for `oss-vdb`, is invoked in `staging.yaml`): 
https://github.com/google/osv.dev/blob/101bed8838b06522e89c6d1c82a88ee11cbf9356/gcp/appengine/deploy.sh#L37
Which is grabbing the commit SHA.

That current version is https://github.com/google/osv.dev/tree/9805438dc8731dabd78c8b3eb6871cb12a5090b5/

Reply:
Thanks for clarifying. Not sure why that commit hash didn't come up with I plugged it into the GitHub search box.